hack/wait-for-kudo.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+attempt_counter=0
+max_attempts=120
+
+while [ "$(kubectl get pod -n kudo-system kudo-controller-manager-0 -o jsonpath='{.status.phase}' || true)" != "Running" ]; do
```

Comment:
not a blocker but we should check by pod conditions till it gets ready. Or directly the `sts` 
`kubectl get sts -n kudo-system kudo-controller-manager -o jsonpath='{.status.readyReplicas}'`

Reply:
Nice! ✅